File: backend/diary-server/src/main/java/com/spuit/maum/diaryserver/domain/Diary.java
package com.spuit.maum.diaryserver.domain;

import com.spuit.maum.diaryserver.domain.common.BaseEntity;
import com.spuit.maum.diaryserver.domain.common.DomainModel;
import javax.persistence.Column;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * Diary 애그리거트 루트 도메인.
 *
 * @author cherrytomato1
 * @version 1.0.0
 */

@DomainModel
@NoArgsConstructor
@Builder
@Getter
@ToString
public class Diary extends BaseEntity {

  @Column(nullable = false)
  String userId;
  String title;
  String content;

}

File: src/frontend/Maum/app/src/main/java/com/spuit/maum/dto/Diary.kt
package com.spuit.maum.dto

import java.io.Serializable
import java.util.*

class Diary(
    var id: Int? = null,
    var title: String? = null,
    var content: String? = null,
    var hashtags: String? = null,
    var date: Date? = null
): Serializable